xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/pvr/ChannelGroups.h

```diff
@@ -0,0 +1,165 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "../../capi/addon-instance/pvr/pvr_channel_groups.h"
+#include "../../AddonBase.h"
```

Comment:
wrong include order

Reply:
Can look but thought not for there, the "C" part is primary for there, so that any problem within becomes seen.